intern-book-app/src/app/pages/list/list.component.html

```diff
@@ -0,0 +1,56 @@
+<div class="header">
+  <mat-toolbar color="primary">
+    <span class="title">{{ title }}</span>
+    <span class="filler"></span
+    ><!--titleと一覧の間を保つパーツ-->
+    <span class="menu-item">一覧</span>
+  </mat-toolbar>
+</div>
+
+<mat-card class="form-card">
+  <mat-card-title>書籍登録</mat-card-title>
+  <mat-card-content>
+    <div class="content">
+      <mat-form-field class="input" appearance="fill">
+        <mat-label>書籍名</mat-label>
+        <input matInput />
+      </mat-form-field>
+
+      <mat-form-field class="input" appearance="fill">
+        <mat-label>あらすじ</mat-label>
+        <textarea matInput></textarea>
+      </mat-form-field>
+
+      <mat-form-field class="input" appearance="fill">
+        <mat-label>評価点</mat-label>
+        <input matInput />
+      </mat-form-field>
+    </div>
+  </mat-card-content>
+
+  <mat-card-actions align="end">
+    <button mat-raised-button color="primary">追加</button>
+  </mat-card-actions>
+</mat-card>
+
+<mat-card class="form-card">
+  <mat-card-title>アンドロイドは電気羊の夢を見るか？</mat-card-title>
+  <mat-card-content
+    >第三次大戦後の未来、サンフランシスコを舞台に賞金稼ぎのリック・デッカードが、火星から逃亡してきた8体のアンドロイドを「処理」するというあらすじ</mat-card-content
+  >
+  <mat-card-content>評価:90点</mat-card-content>
```

Comment:
`<mat-card-content>`はそのcardに一つだけあるのが望ましいですね（多分）
なので以下みたいに書き換えてあげましょう
```suggestion
  <mat-card-content>
    <div>第三次大戦後の未来、サンフランシスコを舞台に賞金稼ぎのリック・デッカードが、火星から逃亡してきた8体のアンドロイドを「処理」するというあらすじ</div>
    <div>評価:90点</div>
  </mat-card-content>
```
表示はcssでいい感じにしてください！
あとどれを使うか迷う`<div>`, `<p>`, `<span>`は[この記事](https://zenn.dev/shu5/articles/e769cdc856b914)がちょっと参考になるかな